src/libraries/Directory.Build.props

```diff
@@ -277,6 +277,10 @@
 
     <TestHostRootPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'testhost', '$(BuildSettings)'))</TestHostRootPath>
 
+    <NETCoreAppRuntimePackPath>$(ArtifactsBinDir)\lib-runtime-packs</NETCoreAppRuntimePackPath>
```

Comment:
nit: please use `$([MSBuild]::NormalizePath())` and `$([MSBuild]::NormalizeDirectory())` if possible in these paths. You'll see a lot of places where we don't use it, but that is because these APIs didn't exist when those were added.

For `RuntimePackTargetFrameworkPack` you might not be able to do so because it is not a full path. These only work with full paths from what I know.